web/app_dev.php

```diff
@@ -12,31 +12,31 @@
 use Symfony\Component\Debug\Debug;
 
 // Access to debug front controllers is only allowed on localhost or with username and password.
-// FIXME: add link to documentation that explains what to do
+// The username and password need to be separated by a colon and then converted to a SHA512 hash.
```

Comment:
It does not matter at all. The string could also be `admin:my:pass:word` as long as the hash can be validated in line 34.